src/lib/assets/Assets.stories.svelte

```diff
@@ -53,9 +55,9 @@
       <div class="flex flex-col items-start justify-start p-4">
         <h2 class="mb-2 text-lg font-semibold">{name}</h2>
         {#if Icon === OpenClose}
-          <input type="checkbox" class="peer hidden" id="mobile-menu-toggle" />
+          <input type="checkbox" id="mobile-menu-toggle" class="peer hidden" bind:checked={isOpen} />
```

Comment:
We should add aria tags to all this at some point, but that should be a separate issue.